invoice/gql/invoice/mutation.py

```diff
@@ -0,0 +1,62 @@
+import logging
+
+import graphene
+from django.contrib.auth.models import AnonymousUser
+from django.core.exceptions import ValidationError
+from graphql import GraphQLError
+from policy.apps import PolicyConfig
+
+from core.gql.gql_mutations.base_mutation import BaseMutation
+from core.schema import OpenIMISMutation
+from invoice.models import Invoice
+from invoice.services import InvoiceService
+
+logger = logging.getLogger(__name__)
+
+
+class GenerateTimeframeInvoices(BaseMutation):
+    _model = Invoice
+    _invoice_service_class: InvoiceService = InvoiceService
+    _mutation_module = 'invoice'
+    _mutation_class = 'GenerateTimeframeInvoices'
+
+    @classmethod
+    def _validate_mutation(cls, user, **data):
+        # TODO: What permissions should be used?
+        if type(user) is AnonymousUser or not user.id or not user.has_perms(
+                PolicyConfig.gql_mutation_create_policies_perms):
```

Comment:
why this perm .gql_mutation_create_policies_perms ?

Reply:
Perms have been changed to the valid ones for this module.